buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -42,13 +42,11 @@ object Versions {
 
   const val io_objectbox: String = "3.5.0"
 
-  const val org_jacoco: String = "0.8.7"
-
-  const val io_mockk: String = "1.12.0"
+  const val io_mockk: String = "1.13.4"
 
   const val android_arch_lifecycle_extensions: String = "1.1.1"
 
-  const val com_android_tools_build_gradle: String = "7.3.1"
```

Comment:
@danielzgtg  We should increase the ```com.android.tools.build:gradle``` version in ```buildSrc``` gradle as well. we should not use two different versions in our project.

Reply:
Good catch!